pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -531,6 +531,9 @@ func (l *LibvirtDomainManager) asyncMigrate(vmi *v1.VirtualMachineInstance, opti
 }
 
 func setupMigration(l *LibvirtDomainManager, vmi *v1.VirtualMachineInstance, dom cli.VirDomain, options *cmdclient.MigrationOptions, loopbackAddress string) (*libvirt.DomainMigrateParameters, error) {
+	l.domainModifyLock.Lock()
```

Comment:
Can you please explain in the commit message why `l.virConn.LookupDomainByName` is not part of the lock (as in other places)?

Reply:
I tried to avoid it in order not to admit we are not solving everything with this.
I am adding the explanation.